documentation.tex

```diff
@@ -0,0 +1,281 @@
+\documentclass[   % inherits article
+  final,          % or
+  % draft,
+  a4paper,        % or
+  % letterpaper   % (etc. pp.)
+  portrait,       % WILL PRODUCE WARNING
+  rscols=3,       % this is the default
+  margin=1.0cm,   % default, or
+  % margin=0.5in, % default
+  titlepage,      % WILL PRODUCE WARNING
+  % twocolumn     % WOULD PRODUCE ERROR
+]{refsheet}
+
+%
+% Adapted from https://tex.stackexchange.com/a/99784.
+%
+
+% Typesetting codeprettier
+\usepackage[dvipsnames]{xcolor}
+\definecolor{lightgray}{gray}{0.95}
+\usepackage{listings}
+\lstdefinestyle{mytex}{
+  language=[LaTeX]TeX,%
+  backgroundcolor=\color{lightgray},%
+  basicstyle=\footnotesize\ttfamily,%
+  keywordstyle=\bfseries,%
+  breaklines=true,%
+  morekeywords={part,chapter,subsection,subsubsection,paragraph,subparagraph}%
+}
+
+\lstset{style=mytex}
+
+% Symbols for SE and GH
+\usepackage{fontawesome} 
+% Symbols for creative commons
+\usepackage{ccicons}
+% linking to the outside world
+\usepackage{hyperref}
+% Martin's weird SE handle
+% (use a different package and Xe/LuaLaTeX if you're doing proper Japanese cheat sheets.)
+\usepackage{CJKutf8} 
+
+% Please no font errors
+% https://tex.stackexchange.com/a/9366/33413
+\newcommand\mybackslash{\char`\\}
```

Comment:
Can you expand on why this is necessary? It probably makes much more sense to create a `\macro` macro (or just use `\verb-\macro-`).

Reply:
the verbatim environments are very fragile, so I'm avoiding them.
Yes, I could have used `\lstinline-\macro-` in the documentation, since I already loaded that package anyway; I might change that. That was basically a workaround as I already completed the documentation, and the only thing was a font warning.